### PR TITLE
All you can eat buffet of debugging guide improvements

### PR DESCRIPTION
**Pull request by PettitWesley**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: troubleshooting/debugging.md [hunk 1]
```diff
@@ -166,9 +175,14 @@ To prevent the error and log loss above, we can set our tail input to have:
...
 
+If you set `Skip_Long_Lines On`, then Fluent Bit will emit the following [error message](https://github.com/fluent/fluent-bit/blob/v2.0.0/plugins/in_tail/tail_file.c#L1319) when it encounters lines longer than its `Buffer_Max_Size`:
+```
+[warn] file=/var/log/app.log have long lines. Skipping long lines.
+```
+
 
 #### Tail Permission Errors
-A common error message for ECS FireLens users who are reading log files is the following:
+A common error message fgor ECS FireLens users who are reading log files is the following:
```
Comment (lubingfeng) [inline troubleshooting/debugging.md]: Spelling.

Comment (Claych) [inline troubleshooting/debugging.md]: It should be an unintentional problem.

Comment (ShelbyZ) [inline troubleshooting/debugging.md]: 🤔

File: troubleshooting/debugging.md [hunk 2]
```diff
@@ -207,6 +221,39 @@ When the input is able to receive logs again, you will see one of the `resume` m
 
 With some inputs, an overlimit warning indicates that you are losing logs- new logs will not be ingested. This is the case with most inputs that stream in data ([forward](https://docs.fluentbit.io/manual/pipeline/inputs/forward) and [TCP](https://docs.fluentbit.io/manual/pipeline/inputs/tcp), for example). If you use [ECS FireLens](https://aws.amazon.com/blogs/containers/under-the-hood-firelens-for-amazon-ecs-tasks/) with Fluent Bit, then the stdout/stderr log input is a forward input and an overlimit warning means that new logs will not be ingested and will be lost. The exception to this is the [tail](https://docs.fluentbit.io/manual/pipeline/inputs/tail) input, which can safely pause and resume without losing logs because it tracks its file offset. When it resumes, it can pick back up reading the file at the last offset (assuming the file was not deleted).
 
+#### How do I tell if Fluent Bit is ingesting and sending data?
+
+*This requires that you [enabled debug logging](#enable-debug-logging*.
```
Comment (ShelbyZ) [inline troubleshooting/debugging.md]: This deeplink is missing closing `)` I think

Comment (PettitWesley) [reply troubleshooting/debugging.md]: thanks!

File: troubleshooting/debugging.md (hunk 1 above)
Comment (PettitWesley) [reply troubleshooting/debugging.md]: Lol oops

Comment (weijianz-opc) [reply troubleshooting/debugging.md]: typo

**Review by lubingfeng — APPROVED** [review]

**Review by weijianz-opc — APPROVED** [review]
LGTM

**Pull request merged**